admin_guide/backup_restore.adoc

```diff
@@ -817,16 +817,15 @@ what path to back up by looking at the `*mountPath*` for volumes in the
 . Get the application data `*mountPath*` from the `*deploymentconfig*`:
 +
 ----
-$ oc export dc/jenkins|grep mountPath
-        - mountPath: /var/lib/jenkins
+$ oc get dc/jenkins -o jsonpath='{ .spec.template.spec.containers[?(@.name=="jenkins")].volumeMounts[?(@.name=="jenkins-data")].mountPath }'
```

Comment:
Looks a little crazy but at the same time you _do_ want to make sure you are looking at the mount path for the correct volume mount on the correct container, as there is no reason we can't add more volume mounts or more containers with volumes in the future to this deployment config.